### DISCUSSION TITLE
Is there a way to set the stroke in XAML?

### DISCUSSION THREAD

**Discussion by ddotlic** [wieslawsoltes/Svg.Skia#137] [Q&A]
Use case I have is hopefully fairly typical - I use SVG files as icons. Thus, there's no styling in the SVG files, none at all.

Is there a way to set the stroke in XAML? I use Avalonia and can use either `Svg` or `Image` control.

**Comment by wieslawsoltes** [comment]
https://wieslawsoltes.github.io/Svg.Skia/articles/xaml/styling-and-previewer/
